index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
```

Comment:
This is probably a better place to have the generic discussion than a particular Chrome bug. @noncombatant, do you definitely think there are some sensors for which a gesture is the right gate, or is it just something for Alex et. al. to consider when looking for reasonable gates for each sensor? My reading of your https://codereview.chromium.org/2847253002/#msg60 indicates the latter.